.vscode/settings.json

```diff
@@ -3,5 +3,8 @@
   "editor.codeActionsOnSave": {
     "quickfix.biome": "explicit"
   },
-  "editor.formatOnSave": true
+  "editor.formatOnSave": true,
```

Comment:
Are these changes needed, or specific to your personal environment?

Reply:
Oups specific to my personnal environnement, should be remove from merge request